### PR TITLE
🐛 Adds microsoft-edge protocol

### PR DESCRIPTION
**Pull request by PHCCorso**
Closes https://github.com/ampproject/amphtml/issues/34167

### CODE REVIEW

File: validator/validator-main.protoascii [hunk 1]
```diff
@@ -1371,6 +1371,7 @@ tags: {
       protocol: "web+mastodon"  # See GitHub issue #14793
       protocol: "wh"
       protocol: "whatsapp"
+      protocol: "microsoft-edge"
```
Comment (Gregable) [inline validator/validator-main.protoascii:1374]: Please add a comment with a link to:
https://docs.microsoft.com/en-us/windows/uwp/launch-resume/launch-default-app#microsoft-edge-uri-scheme

or something similar.

Comment (PHCCorso) [reply validator/validator-main.protoascii:1375]: @Gregable Sure. 

Added it :+1: .

**Review by Gregable — APPROVED** [review]

**Comment by honeybadgerdontcare** [comment]
@alanorozco could you triage or review, Will is no longer on AMP and unsure who owns `amp-bind`

File: validator/validator-main.protoascii (hunk 1 above)
Comment (alanorozco) [reply validator/validator-main.protoascii:1375]: Looks the validation entry needs to be sorted alphabetically as well.

File: extensions/amp-bind/0.1/bind-validator.js [hunk 2]
```diff
@@ -389,6 +389,7 @@ function createElementRules_() {
           'web+mastodon': true,
           'wh': true,
           'whatsapp': true,
+          'microsoft-edge': true,
```
[the remark this answers is not in the document]
Comment (PHCCorso) [reply extensions/amp-bind/0.1/bind-validator.js:392]: @alanorozco Yup, fixed. :+1:

[the remark this answers is not in the document]
Comment (alanorozco) [reply extensions/amp-bind/0.1/bind-validator.js:392]: You need to sort it on this file as well.

**Comment by alanorozco** [comment]
@phcorso You need to sort it on `bind-validator.js` as well.

File: extensions/amp-bind/0.1/bind-validator.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (PHCCorso) [reply extensions/amp-bind/0.1/bind-validator.js]: @alanorozco silly me. Totally overlooked that. Fixed.

[the remark this answers is not in the document]
Comment (PHCCorso) [reply extensions/amp-bind/0.1/bind-validator.js]: @alanorozco ?

**Review by alanorozco — APPROVED** [review]

**Pull request merged**